WA.Pizza.Infrastructure/Services/Mapster/MappingConfig.cs

```diff
@@ -29,18 +29,19 @@ public static void Configure()
 
 			TypeAdapterConfig<CatalogItem, ListCatalogItemsDTO>
 				.NewConfig()
+				.Map(dest => dest.Id, src => src.Id)
 				.Map(dest => dest.Name, src => src.Name)
 				.Map(dest => dest.Price, src => src.Price.ToString("0.00"))
 				.Map(dest => dest.PictureBytes, src => src.PictureBytes)
 				.Map(dest => dest.StorageQuantity, src => src.StorageQuantity);
 
-			TypeAdapterConfig<CatalogItem, CatalogItemDTO>
-				.NewConfig()
-				.Ignore(dest => dest.StorageQuantity)
-				.Map(dest => dest.Id, src => src.Id)
-				.Map(dest => dest.Name, src => src.Name)
-				.Map(dest => dest.Price, src => src.Price.ToString("0.00"))
-				.Map(dest => dest.PictureBytes, src => src.PictureBytes);
+			//TypeAdapterConfig<CatalogItem, CatalogItemDTO>
+			//	.NewConfig()
+			//	.Ignore(dest => dest.StorageQuantity)
+			//	.Map(dest => dest.Id, src => src.Id)
+			//	.Map(dest => dest.Name, src => src.Name)
+			//	.Map(dest => dest.Price, src => src.Price.ToString("0.00"))
+			//	.Map(dest => dest.PictureBytes, src => src.PictureBytes);
```

Comment:
Why?

Reply:
I combined Add and Update in CatalogDataService so I don't use this mapping anymore.